WarehouseManagement.Core/Services/MarkerService.cs

```diff
@@ -100,36 +100,36 @@ public async Task<IEnumerable<MarkerDto>> GetAllAsync()
 
     public async Task<MarkerDto?> GetByIdAsync(int id)
     {
-        return await repository
-            .AllReadOnly<Marker>()
-            .Where(m => m.Id == id)
-            .Select(m => new MarkerDto
-            {
-                Name = m.Name,
-                Deliveries = m
+        Marker marker = (await repository.GetByIdAsync<Marker>(id))!;
```

Comment:
from what I understood in the comments for #13 , we wont be needing this change?